generators/generatorPickerFunctions.h

```diff
@@ -111,6 +112,26 @@ namespace rpwa {
 
 	};
 
+
+	/**
+	 * Class to pick a m and t' uniformly in a given range
+	 */
+	class uniformMassAndTPicker : public massAndTPrimePicker {
+
+	  public:
+
+		uniformMassAndTPicker(){}
+		uniformMassAndTPicker(const uniformMassAndTPicker& picker): massAndTPrimePicker(picker){}
```

Comment:
I don't think you actually need to provide these two constructors, the default should be fine